README.md

```diff
@@ -1,48 +1,31 @@
-![cf](http://i.imgur.com/7v5ASc8.png) Linked List Data Structure
-====
-
-## To Submit this Assignment
-  * Do all your work in a public repository called `data-structures-and-algorithms`
-  * fork this repository
-  * write all of your code in a directory named `linked-lists`
-  * push to your repository
-  * submit a pull request to this repository
-  * submit a link to your PR in canvas
-  * write a question and observation on canvas
-
-## Requirements  
-#### Configuration  
-  <!-- list of files, configurations, tools, etc that are required -->
-  Your lab directory must include  
-  * **index.js** -- The entry point to your module
-  * **README.md** -- with a documentation about your lab
-  * **.gitignore** -- with a robust .gitignore
-  * **.eslintrc** -- with the class .eslintrc file
-  * **.eslintignore** -- with the class .eslintignore
-  * **.package.json** -- with all dependencies and dev-dependencies
-  * **lib/** -- directory for holding your programs helper modules
-  * **test/** -- directory for holding your programs unit and integration tests
-
-#### Testing  
-  * write at least three test assertions for each method of the Singly Linked List Data Structure
-  * organize your tests into appropriate describe/it blocks for test output readability
-
-####  Documentation  
-  * in your README, write documentation for you data structures
-  * your documentation should includes code block useage examples
-  * provide instructions for:
-    * installing and using your data structure
-    * accessing each method
-    * running your tests
-
-#### Feature Tasks  
-* implement a Singly Linked List (SLL) data structure
+# 02-Linked LIsts
+
+Link to Master: https://github.com/jbach197/05-linked-lists
+Link to Pull Request: https://github.com/jbach197/05-linked-lists/compare/master...jbach197:lab05-jen?expand=1
+Link to Travis: [![Build Status](https://travis-ci.com/jbach197/05-linked-lists.svg?branch=lab05-jen)](https://travis-ci.com/jbach197/05-linked-lists)
+
+# Installation Instructions
+1) Clone the repo
+2) Run npm i to install the dependencies
+3) Run npm test to complete the tests
+4) Run npm lint for the linter
+
+#Requirements
```

Comment:
Make sure to include a space between the '#' and the 'Requirements' - if there isn't a space it will display as '#Requirements' rather than **Requirements**